clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/SearchDAO.scala

```diff
@@ -39,8 +41,12 @@ trait SearchDAO {
     * @param index    The index in which to update the document.
     * @tparam D The type of the document.
     */
-  def updateMetadata[D <: ClioDocument](document: D)(
+  final def updateMetadata[D <: ClioDocument](document: D)(
     implicit index: ElasticsearchIndex[D]
+  ): Future[Unit] = updateMetadata(document.asJson(index.encoder))
```

Comment:
sometimes we import index.decoder or encoder and sometimes it is being passed in as implicit param. I'm curious why we do it differently at times?

Reply:
My idea was to import if it's used in > 1 spot, and otherwise just pass it explicitly, but it looks like all usages are just for 1 spot. I'll make it consistent.